keanu-python/keanu/sequence.py

```diff
@@ -31,6 +31,9 @@ def add(self, vertex: Vertex, label: Optional[str] = None) -> None:
     def get(self, label: str) -> Vertex:
         return Vertex._from_java_vertex(self.unwrap().get(_VertexLabel(label).unwrap()))
 
+    def get_contents(self) -> Dict[str, Vertex]:
+        return self.unwrap().getContents()
```

Comment:
You'll probably have to convert this to python map (of str to python Vertex)? Mypy is probably not complaining because there are no tests for it.

Reply:
+1